sr_robot_launch/test/test_sr_ur_unlock.py

```diff
@@ -0,0 +1,171 @@
+#!/usr/bin/env python
+
+# Copyright 2021 Shadow Robot Company Ltd.
+#
+# This program is free software: you can redistribute it and/or modify it
+# under the terms of the GNU General Public License as published by the Free
+# Software Foundation version 2 of the License.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License for
+# more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program. If not, see <http://www.gnu.org/licenses/>.
+
+import rospy
+import rostest
+from sr_robot_commander.sr_hand_commander import SrHandCommander
+from actionlib_msgs.msg import GoalStatusArray
+from unittest import TestCase
+from std_msgs.msg import Bool
+from std_srvs.srv import Trigger
+from ur_dashboard_msgs.srv import IsProgramRunning
+from sr_robot_launch.sr_ur_arm_unlock import SrUrUnlock
+from sr_robot_launch.mock_sr_ur_robot_hw import MockUrRobotHW
+from ur_dashboard_msgs.msg import SafetyMode, ProgramState, RobotMode
+import sys
+
+
+class CommonTests:
+    def arm_setup(self, side):
+        self.press_pedal()
+        self.assertTrue(self.get_program_running(side))
+
+    def arm_mock(self, side):
```

Comment:
why is this names arm_mock?

Reply:
it's testing that the mock dashboard server is working (i.e. has it been initialised to `POWER_OFF`). renamed here: 7d5875d925ab75619ae25a0f66d0afb5446a0f42